config/homebin/vagrant_provision

```diff
@@ -65,6 +68,6 @@ else
 	echo -e "${RED} ⚠ The failed provisioners left these logs:"
 
 	# List failed provisioners, and pipe them through xargs to print them in a formatted way.
-	ls -A /vagrant/failed_provisioners/ | xargs -I "LOGFILE" echo -e "   - log/provisioners/${date_time}/LOGFILE.log"
+	ls -A /vagrant/failed_provisioners/ | xargs -I "LOGFILE" vvv_error "   - <b>log/provisioners/${date_time}/LOGFILE.log</b>"
```

Comment:
this is an html tag, why on a log file?

Reply:
Ok I saw in another file that will be replaced, maybe add something in our doc website?

Reply:
At the moment we don't document any of these functions so we'd want to figure out a solution for that, something like PHPDoc for shell scripts

Reply:
https://github.com/reconquest/shdoc There is this that is for bash, maybe is for another ticket?

Reply:
That looks like a good option :) Can you make a new ticket for it?